Setup/UpgradeSchema.php

```diff
@@ -230,4 +227,30 @@ private function updateWebhookLogTable($setup)
             ]
         );
     }
+
+    private function setupExternalCustomerEntityTable($setup)
+    {
+        $tableCreated = $setup->getConnection()->isTableExists('external_customer_entity');
+        if ($tableCreated) {
+            return;
+        }
+
+        $table = $setup->getConnection()
+            ->newTable($setup->getTable('external_customer_entity'))
+            ->addColumn(
```

Comment:
can you make this reference the `customer` table ID? otherwise this is fine.

Reply:
oh this is jsut the primary key column i'll add a new column for customer_id

Reply:
ok cool